MekHQ/src/mekhq/campaign/universe/Planet.java

```diff
@@ -642,36 +582,15 @@ public String getHPGClass(DateTime when) {
         return StarUtil.getHPGClass(getHPG(when));
     }
 
-    public Integer getPopulationRating(DateTime when) {
-        return getEventData(when, populationRating, new EventGetter<Integer>() {
-            @Override public Integer get(PlanetaryEvent e) { return e.populationRating; }
-        });
-    }
-    
-    public String getPopulationRatingString(DateTime when) {
-        Integer pops = getPopulationRating(when);
-        return (null != pops) ? StarUtil.getPopulationRatingString(pops.intValue()) : "unknown";
-    }
-    
-    public String getGovernment(DateTime when) {
-        return getEventData(when, government, new EventGetter<String>() {
-            @Override public String get(PlanetaryEvent e) { return e.government; }
+    public Long getPopulation(DateTime when) {
```

Comment:
```suggestion
    public long getPopulation(DateTime when) {
```

Reply:
I think this one actually needs to be Long because the underlying EventData uses a Long for population because planets that are not populated get null values.